pkg/scheduler/scheduler.go

```diff
@@ -499,10 +499,11 @@ func (sched *Scheduler) scheduleOne() {
 		// into the resources that were preempted, but this is harmless.
 		if fitError, ok := err.(*core.FitError); ok {
 			preemptionStartTime := time.Now()
-			sched.preempt(pod, fitError)
-			metrics.PreemptionAttempts.Inc()
-			metrics.SchedulingAlgorithmPremptionEvaluationDuration.Observe(metrics.SinceInMicroseconds(preemptionStartTime))
-			metrics.SchedulingLatency.WithLabelValues(metrics.PreemptionEvaluation).Observe(metrics.SinceInSeconds(preemptionStartTime))
+			if _, err := sched.preempt(pod, fitError); err == nil {
+				metrics.PreemptionAttempts.Inc()
```

Comment:
preemption attempts must be incremented regardless of the error. It has been an attempt after all. Some attempts may fail.

Reply:
Oh, I see, so we should leave the !util.PodPriorityEnabled() || sched.config.DisablePreemption judgment outside of preempt, at this time, preemption did not occur. 
Thanks for your explanation, I have revised it.